guides/04_Quality_Bandwidth_Connectivity.md

```diff
@@ -80,7 +80,7 @@ When possible, profile on devices that have similar performance characteristics
 
 #### Choose a lower local video quality
 
-Sometimes it is better to sacrifice video quality in order to prioritize audio. You can call [chooseVideoInputQuality](https://aws.github.io/amazon-chime-sdk-js/interfaces/devicecontroller.html#choosevideoinputquality)(width, height, frameRate, maxBandwidthKbps) and lower the maximum bandwidth in real-time. You can also adjust the resolution and frame rate if you call the method before starting local video (or stop and then restart the local video). See the section below on values you can use for `chooseVideoInputQuality`.
+Sometimes it is better to sacrifice video quality in order to prioritize audio. You can call [chooseVideoInputQuality(width, height, frameRate)](https://aws.github.io/amazon-chime-sdk-js/interfaces/audiovideofacade.html#choosevideoinputquality) and [setVideoMaxBandwidthKbps(maxBandwidthKbps)](https://aws.github.io/amazon-chime-sdk-js/interfaces/audiovideofacade.html#setvideomaxbandwidthkbps) to lower the maximum bandwidth in real-time. You can also adjust the resolution and frame rate if you call the method before starting local video (or stop and then restart the local video). See the section below on values you can use for `chooseVideoInputQuality`.
```

Comment:
The `chooseVideoInputQuality` API now belongs to `DeviceoControllerFacade`, could you update the link to:
https://aws.github.io/amazon-chime-sdk-js/interfaces/devicecontrollerfacade.html#choosevideoinputquality

Reply:
Thanks updated.